reference/price_lists.v3.yml

```diff
@@ -2692,6 +2717,31 @@ paths:
         required: true
         schema:
           type: string
+  "/pricelists/{price_list_id}/assignments":
+    put:
+      tags:
+        - Price Lists Assignments
+      description: Upsert a single `Price List Assignment` for a `Price List`.
+      summary: Upsert Price List Assignment
+      operationId: upsertPriceListAssignment
+      parameters:
+        - $ref: "#/components/parameters/PriceListIdParam"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AssignmentForPutRequest"
+        required: true
+      responses:
```

Comment:
Testing this with a `channel_id` of `0` or a negative number returns a `502` error:
```json
{
    "status": 502,
    "title": "Bad Gateway",
    "type": "https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes",
    "errors": {}
}
``` 
We don't need to change anything, but you might want to report it as a bug to the engineering team.